op-dispute-mon/mon/forecast.go

```diff
@@ -85,6 +86,10 @@ func (f *Forecast) forecastGame(game *monTypes.EnrichedGameData, metrics *foreca
 		if metrics.LatestInvalidProposal < game.Timestamp {
 			metrics.LatestInvalidProposal = game.Timestamp
 		}
+	} else {
+		if metrics.LatestValidProposal < game.Timestamp {
+			metrics.LatestValidProposal = game.Timestamp
+		}
```

Comment:
This won't tell us if the _proposer_ specifically created a valid proposal but if there are valid proposals being created frequently enough in general. If we want to alert against the proposer's actions here we would need to check that the game was created by the proposer.

Metricing this is valuable still, just wanted to call out it doesn't provide the specified alerting.

Reply:
Yeah good call out -  this is a useful thing to monitor, but we should monitor the proposer itself based on the proposer's own metrics of when it published.  There's some discussion of that in the referenced ticket and this is explicitly not marking it as fixed.